cli/tests/testdata/jsr/version_not_found/main.out

```diff
@@ -1,4 +1,5 @@
 Download http://127.0.0.1:4250/@denotest/deps/meta.json
+Download http://127.0.0.1:4250/@denotest/deps/meta.json
 error: Could not find constraint in the list of versions: @denotest/deps@0.1.4
   Specifier: jsr:@denotest/deps@0.1.4/mod.ts
     at file:///[WILDCARD]/version_not_found/main.ts:1:19
```

Comment:
A very minor issue is now we request the metadata twice even if the metadata was not originally loaded from the cache. Fixing this would require telling deno_graph about if a specifier was loaded from the cache or not, which I think would be too much complexity and isn't worth fixing.